pages/test_analytics/importing_json.md.erb

```diff
@@ -137,6 +137,7 @@ Schematically, the JSON test results data is like this:
    - [Test result](#json-test-results-data-reference-test-result-objects)
       * [History](#json-test-results-data-reference-history-objects)
          - [Spans](#json-test-results-data-reference-span-objects)
+             * [Detail](#json-test-results-data-reference-detail-objects)
```

Comment:
This doesn't seem to render correctly. Any thoughts?

![Screenshot from 2022-08-15 09-31-11](https://user-images.githubusercontent.com/1182969/184555823-81db72d6-0569-4183-90ef-700523d87cd8.png)

Reply:
They
    render
          according
               to your indentation :)

Reply:
Seems like 5 levels is not working. Removing the top level "a test result array of" solves the problem.